### ISSUE TITLE
--version reports 2.3

### ISSUE THREAD

**Issue by om26er** [lxc/lxd#2460]
I have lxd 2.4 installed on my system but lxd --version (or lxc --version) returns 2.3

```
lxd:
  Installed: 2.4-0ubuntu1
  Candidate: 2.4-0ubuntu1
  Version table:
 *** 2.4-0ubuntu1 500
        500 http://mirrors.nayatel.com/ubuntu yakkety/main amd64 Packages
        100 /var/lib/dpkg/status
```

**Comment by tych0** [comment]
Is there a stray one in your path somewhere? What's the output of `which -a`? I just looked at the code and it *should* be right, but who knows what that means :)

**Comment by stgraber** [comment]
Crap, we do have a problem with the LXD 2.4 tarball saying 2.3... Will release a 2.4.1 now to fix that mess.

**Comment by stgraber** [comment]
Upstream tag is correct, upstream published tarball, not so much...

**Comment by stgraber** [comment]
Fixed, 2.4.1 released.
